sdk/management/pipelines/fluentJavaMgmt.yml

```diff
@@ -1,14 +1,57 @@
 # External variables Settable at queue time:
 # "pomFile" : "pom.xml".   This is the project object model file for Maven.
 
+
 trigger:
-  - vnext
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - sdk/managment/azure/
+      - sdk/management/samples/
+      - sdk/management/pom.xml
+      - sdk/resources/mgmt/
+      - sdk/storage/mgmt/
+      - sdk/keyvault/mgmt/
+      - sdk/managedserviceidentity/mgmt/
+      - sdk/authorization/mgmt/
+      - sdk/network/mgmt/
+      - sdk/compute/mgmt/
+      - sdk/containerregistry/mgmt/
+      - sdk/dns/mgmt/
+      - sdk/containerservice/mgmt/
+      - sdk/cosmos/mgmt/
+      - sdk/appservice/mgmt/
+      - sdk/monitor/mgmt/
+      - sdk/sql/mgmt/
 
 pr:
-  - vnext
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - sdk/managment/azure/
```

Comment:
spelling

or you can just add `sdk/management`?

Reply:
I have thought about it. And since it container several useless file in pipeline, so I include every important file individually

Reply:
Done